tutorials/actors_demo/actors_demo.md

```diff
@@ -0,0 +1,169 @@
+
+We can make our actor follows the specified trajectory forever and to start playing as soon as the world is loaded as follows:
+
+```xml
+<script>
+    <loop>true</loop>
+    <delay_start>0.000000</delay_start>
+    <auto_start>true</auto_start>
+```
+
+The script is defined between the `<actor>` `</actor>` tags.
+
+Inside the `<script>` tag the following parameters are available:
+
+* `loop`: Set this to true for the script to be repeated in a loop. For a fluid continuous motion, make sure the last waypoint matches the first one, as we'll do below.
+
+* `delay_start`: This is the time in seconds to wait before starting the script. If running in a loop, this time will be waited before starting each cycle.
+
+* `auto_start`: Set to true if the animation should start as soon as the simulation starts playing. It is useful to set this to false if the animation should only start playing only when triggered by a plugin, for example.
+
+Let's define the trajectory as a sequence of waypoints:
+
+```xml
+        <trajectory id="0" type="walk">
+            <waypoint>
+                <time>0</time>
+                <pose>0 0 1.0 0 0 0</pose>
+            </waypoint>
+            <waypoint>
+                <time>2</time>
+                <pose>2.0 0 1.0 0 0 0</pose>
+            </waypoint>
+            <waypoint>
+                <time>2.5</time>
+                <pose>2 0 1.0 0 0 1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>4</time>
+                <pose>2 2 1.0 0 0 1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>4.5</time>
+                <pose>2 2 1.0 0 0 3.142</pose>
+            </waypoint>
+            <waypoint>
+                <time>6</time>
+                <pose>0 2 1 0 0 3.142</pose>
+            </waypoint>
+            <waypoint>
+                <time>6.5</time>
+                <pose>0 2 1 0 0 -1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>8</time>
+                <pose>0 0 1.0 0 0 -1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>8.5</time>
+                <pose>0 0 1.0 0 0 0</pose>
+            </waypoint>
+        </trajectory>
+    </script>
+</actor>
+```
+
+Within the `<trajectory>` tag we can define a series of waypoints which our actor will follow. The `<trajectory>` has two attributes: `id` and `type`. The `type` should have the same name as the animation `walk`. The `trajectory` parameters are as follow:
+
+* `waypoint`: There can be any number of waypoints in a trajectory. Each waypoint consists of a time and a pose:
+    * `time`: The time in seconds, counted from the beginning of the script, when the pose should be reached.
+    * `pose`: The pose which should be reached.
```

Comment:
This has changed a bit from Gazebo to Ignition, so I think it's worth it elaborating a bit more. The pose here will be with respect to the initial `<actor><pose>` (on classic, this pose is always global, which is inconvenient sometimes).